process/connections.proto_builder.go

```diff
@@ -1122,6 +1133,34 @@ func (x *Connection_DnsStatsByDomainOffsetByQueryTypeEntryBuilder) SetValue(cb f
 	x.writer.Write(x.buf.Bytes())
 }
 
+type Connection_TcpFailuresByErrCodeEntryBuilder struct {
+	writer  io.Writer
+	buf     bytes.Buffer
+	scratch []byte
+}
+
+func NewConnection_TcpFailuresByErrCodeEntryBuilder(writer io.Writer) *Connection_TcpFailuresByErrCodeEntryBuilder {
+	return &Connection_TcpFailuresByErrCodeEntryBuilder{
+		writer: writer,
+	}
+}
+func (x *Connection_TcpFailuresByErrCodeEntryBuilder) Reset(writer io.Writer) {
+	x.buf.Reset()
+	x.writer = writer
+}
+func (x *Connection_TcpFailuresByErrCodeEntryBuilder) SetKey(v int32) {
+	x.scratch = x.scratch[:0]
+	x.scratch = protowire.AppendVarint(x.scratch, 0x8)
```

Comment:
What is `0x8` here? In general, I think it'd help readability if the constants were set to named variables. Do they ever change?